dask_sql/context.py

```diff
@@ -501,6 +504,30 @@ def create_schema(self, schema_name: str):
         """
         self.schema[schema_name] = SchemaContainer(schema_name)
 
+    def alter_schema(self, old_schema_name, new_schema_name):
+        """
+        Alter schema
+
+        Args:
+             old_schema_name:
+             new_schema_name:
+        """
+        self.schema[new_schema_name] = copy.deepcopy(self.schema[old_schema_name])
+        del self.schema[old_schema_name]
```

Comment:
Could we avoid the deep copy here by just popping the original schema?

```suggestion
        self.schema[new_schema_name] = self.schema.pop(old_schema_name)
```